Profile Folder/chrome/locale/ja/properties/gsettings.properties

```diff
@@ -1,15 +1,15 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Geckium の設定
 
-addAnApp=Add an app
+addAnApp=アプリを追加する
 
-themeHasNoDescription=This theme has no description.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
-showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
-apply=[!!! Âƥƥℓ¥ !!!]
-yes=[!!! Ýèƨ !!!]
+themeHasNoDescription=このテーマには概要がありません。
+builtInTheme=ビルトインテーマ
```

Comment:
`builtInTheme` seems to be `Built-in theme`, I think it should probably just be `[BUILT-IN]`.
Here's another example of how it looks like in english:
![image](https://github.com/user-attachments/assets/7394a0ab-f886-46b8-b516-88410df1fa2a)